docs/api/conventions.md

```diff
@@ -461,6 +461,8 @@ export default function SomeRouteComponent() {
 
 ### `loader`
 
+<docs-success>Watch the <a href="https://www.youtube.com/playlist?list=PLXoynULbYuEDG2wBFSZ66b85EIspy3fy6">📼 Remix Single</a>: <a href="https://www.youtube.com/watch?v=NXqEP_PsPNc&list=PLXoynULbYuEDG2wBFSZ66b85EIspy3fy6">Loading data into components</a></docs-success>
```

Comment:
FYI, I added `<docs-success>` because links in `<docs-info>` don't look great. So here's what `<docs-success>` looks like: 

![image](https://user-images.githubusercontent.com/1500684/151462196-f99de2ff-972d-436f-9c92-4266cbc2ed63.png)

![image](https://user-images.githubusercontent.com/1500684/151462424-b3f22e44-7513-48de-b262-2fb4a2c068df.png)